converting/infixToPostfix.c

```diff
@@ -0,0 +1,168 @@
+#include<stdio.h> 
+#include<stdlib.h> 
+#define max 30 
+char stack[max]; 
+
+int t=-1; 
+int flag=0; 
+void push(char val ) 
+{ 
+  if(t==max-1) 
+  { 
+   printf("overflow\n"); 
+  } 
+  else 
+  { 
+   t++; 
+   stack[t]=val; 
+    
+  } 
+
+} 
+char pop() 
+{ char n; 
```

Comment:
Please use this [code style conventions](https://www.gnu.org/prep/standards/html_node/Formatting.html#Formatting) for formatting your code.